src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.WriteEntry.File.Tests.cs

```diff
@@ -206,5 +208,84 @@ public void Add_SymbolicLink(TarEntryFormat format, bool createTarget)
                 Assert.Null(reader.GetNextEntry());
             }
         }
+
+        [Theory]
+        [MemberData(nameof(WriteEntry_CopyArchive_Data))]
+        public void WriteEntry_CopyArchive(TestTarFormat testFormat, string testCaseName, CompressionMethod compressionMethod, bool copyData)
+        {
+            TarEntryFormat entryFormat = GetEntryFormatForTestTarFormat(testFormat);
+            string pathWithExpectedFiles = GetTestCaseUnarchivedFolderPath(testCaseName);
```

Comment:
Not related to this PR but why does this method uses GetCurentDirectory?

Reply:
The dotnet process runs inside the root folder that contains the test assets. When you run the tests on the command line, you can see the folder path:

```
  ----- start Wed 07 Sep 2022 08:12:13 PM PDT =============== To repro directly: =====================================================
  pushd /home/carlos/runtime/artifacts/bin/System.Formats.Tar.Tests/Debug/net7.0-unix                   <- ***THIS FOLDER HERE***
  /home/carlos/runtime/artifacts/bin/testhost/net7.0-Linux-Debug-x64/dotnet exec --runtimeconfig System.Formats.Tar.Tests.runtimeconfig.json --depsfile System.Formats.Tar.Tests.deps.json xunit.console.dll System.Formats.Tar.Tests.dll -xml testResults.xml -nologo -notrait category=OuterLoop -notrait category=failing
```